README.md

```diff
@@ -1 +1,7 @@
-# My-playground
\ No newline at end of file
+# My-playground
+
+This is a place to learn markdown
```

Comment:
Please put a period at the end of your sentence.

Reply:
Done!